src/Makefile.am

```diff
@@ -94,6 +99,23 @@ cockpit-bridge.pyz: $(MANIFESTS) $(PYTHON_BRIDGE_FILES) $(SYSTEMD_CTYPES_STAMP)
 	$(AM_V_GEN) python3 -m zipapp --python /usr/bin/python3 --output $@ --main cockpit.bridge:main tmp/pyz
 	@chmod a+x $@
 
+CLEANFILES += injector.pyz
+injector.pyz: $(MANIFESTS) $(PYTHON_BRIDGE_FILES) $(SYSTEMD_CTYPES_STAMP)
+	@rm -rf tmp/bundle-pyz
+	@mkdir -p tmp/bundle-pyz tmp/bundle-pyz/dist/ssh tmp/bundle-pyz/dist/shell
+	@cp -r $(srcdir)/src/cockpit src/systemd_ctypes/ tmp/bundle-pyz
+	@cp -r $(srcdir)/pkg/ssh/manifest.json tmp/bundle-pyz/dist/ssh
+	@cp -r $(srcdir)/pkg/shell/manifest.json tmp/bundle-pyz/dist/shell
```

Comment:
I'm afraid I need some more background/explanation around what this does. As I see it, we currently have a ~ 12 MB cockpit-bridge.pyz (which bundles all the pages), and then an additional injector.pyz which contains another copy of the bridge code *and* another copy of the shell.

As a draft this is fine of course, but I'd like to understand the eventual plan here. cockpit-bridge.pyz  should not contain any webpack, right? These should be served either from XDG_DATA_DIRS on the target system, or from cockpit-ws. So bridge.pyz is going to be small-ish. Could it not just include the bootloader code? Or can the injector *only* contain the boot loader and fetch bridge.pyz from ws then? But if so, why does it make sense to have two zipapps and that additional logic, instead of just one?

Reply:
I think my idea going forward is something like this:

 - `cockpit-bridge.pyz`, as it is, should go away
 - installing the `cockpit-bridge` package on a server, as we do today, should look more like a traditional python application, with files installed into `site-packages` or whatever other distro-specific place is appropriate
 - running the `pycheck` tests in-tree should also run against the files directly.  Something like this works:
```
PYTHONPATH=src ./test-server python3 -m cockpit.bridge
```
 - we will want to produce the injector bundle for some specific use-cases:
   - the -ws container might want to inject this to machines that it connects to over SSH
   - ditto Cockpit Client